requirements.txt

```diff
@@ -1,6 +1,5 @@
-zeroc-ice==3.6.5
-omero-py==5.10.0
-pandas==1.1.5
-numpy==1.19.5
-ezomero>=0.3.1
-openpyxl==3.0.9
\ No newline at end of file
+pandas>=1.1.5
+numpy>=1.22.0,<2.0.0
+ezomero>=2.1.0
```

Comment:
Conflicting dependencies: omero-cli-transer 0.7.3 requires ezomero 2.0.0, this requirements file requires ezomero>=2.1.0